csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1516,6 +1728,22 @@ Valid values are:
     not_contacted
     open
 
+Each status is mutually exclusive - only one status is valid for a particular vulnerability at a particular time. As the vulnerability ages, a party’s involvement could move from state to state. However, in many cases, a document producer may choose not to issue CSAF documents at each state, or simply omit this element altogether. It is recommended, however, that vendors that issue CSAF documents indicating an open or in-progress Involvement should eventually expect to issue a document as `disputed` or `completed`.
+
+> The two vulnerability involvement status states, `contact_accepted` and `not_contacted` are intended for use by document producers other than vendors (such as research or coordinating entities).
+
+The value `completed` indicates that the vendor asserts that investigation of the vulnerability is complete. No additional information, fixes, or documentation from the vendor about the vulnerability should be expected to be released.
+
+The value `contact_accepted` indicates that the document producer attempted to contact the affected vendor.
+
+The value `disputed` indicates that the vendor disputes the vulnerability report in its entirety. Vendors should indicate this status when they believe that a vulnerability report regarding their product is completely inaccurate (that there is no real underlying security vulnerability) or that the technical issue being reported has no security implications.
```

Comment:
It is unclear to me how this value should be used. I would assume that `disputed` in the context of the party means, that the named party and its affiliation to the vulnerability is disputed (and not the vulnerability report in its entirety).
If there is no security implication, the vulnerability CVSS score would indicate this by a value of 0.
Is there any case, where the value `disputed` would be used? At least from a vendor perspective, this value will never be useful as disputed or non-responding parties will not be mentioned in the advisory.

Reply:
Defining categories in deeply nested structures sometimes makes us miss the forrest for the trees - I will read CVRF v1.2 again to see how we did there. Thanks for bringing this scope / use question up @tolim

Reply:
This is now tracked as #220. The solution will be probably part of one of the next editor revisions.